pkgs/development/compilers/graalvm/community-edition/buildGraalvm.nix

```diff
@@ -109,6 +109,9 @@ let
     installCheckPhase = ''
       runHook preInstallCheck
 
+      echo "Testing Jshell"
+      echo '1 + 1' | $out/bin/jshell
```

Comment:
Testing if it returns 2 would be cool!

Reply:
This would be a fairly fragile test (the output of `jshell` can change in the future). The only important thing here is to make `jshell` exit successfully.